bindings/java/src/executor.rs

```diff
@@ -186,10 +188,11 @@ pub(crate) fn executor_or_default<'a>(
 ///
 /// This function could be only when the lib is loaded.
 unsafe fn default_executor<'a>(env: &mut JNIEnv<'a>) -> Result<&'a Executor> {
-    RUNTIME.get_or_try_init(|| {
+    Ok(RUNTIME.get_or_init(|| {
         make_tokio_executor(
             env,
             available_parallelism().map(NonZeroUsize::get).unwrap_or(1),
         )
-    })
+        .expect("Failed to initialize default executor")
```

Comment:
When handling panic in bindings, we should be very careful to avoid spreading panic across language boundaries. The bindings code is designed this way intentionally. It is recommended not to change returned errors into `expect` or `panic`.

Reply:
/// This function could be only when the lib is loaded.
since we have this comment I wonder its ok, because maybe it is load time things, if error happens means this is not load?

Reply:
> since we have this comment I wonder its ok, because maybe it is load time things, if error happens means this is not load?

Yes, this is exactly why we need errors. A panic here could completely break the user's application if they haven't protected against it. OpenDAL will become part of the user's own business logic, and no user will rely solely on OpenDAL. We must ensure that we do not break their applications.

Reply:
copy will fix it today